test/fuzz/CMakeLists.txt

```diff
@@ -33,17 +22,42 @@ function(add_fuzz_test name label)
                 XPTI_SUBSCRIBERS=$<TARGET_FILE:collector>
                 UR_ENABLE_LAYERS=UR_LAYER_TRACING)
     endif()
+
+    if(UR_BUILD_ADAPTER_L0 OR UR_BUILD_ADAPTER_ALL)
+        list(APPEND ENV_VARS
+                UR_ADAPTERS_FORCE_LOAD=\"$<TARGET_FILE:ur_adapter_level_zero>\")
+        if(UR_USE_ASAN)
+            list(APPEND ENV_VARS
+                    NEOReadDebugKeys=1
+                    DisableDeepBind=1)
+        endif()
+    else()
+        list(APPEND ENV_VARS UR_ADAPTERS_FORCE_LOAD=\"$<TARGET_FILE:ur_adapter_null>\")
+    endif()
+    
+    add_test(NAME ${TEST_TARGET_NAME}
+        COMMAND fuzztest-base ${ARGN} -verbosity=1 -detect_leaks=0
```

Comment:
is this `-detect_leaks=0` needed here? it's set for ASAN separately

Reply:
It's required only here. I've removed the env var now.